packages/pn-personafisica-webapp/src/navigation/__test__/navigation.utility.test.ts

```diff
@@ -1,11 +1,13 @@
 import { vi } from 'vitest';
+
 import { EventPageType } from '@pagopa-pn/pn-commons';
 
 import { getConfiguration } from '../../services/configuration.service';
 import { getCurrentEventTypePage, goToLoginPortal } from '../navigation.utility';
 import { APP_STATUS, DELEGHE, DETTAGLIO_NOTIFICA, NOTIFICHE, RECAPITI } from '../routes.const';
 
 const replaceFn = vi.fn();
+window.open = vi.fn();
```

Comment:
Instead to do this, copy the code from Footer.test.tsx. Do this for all tests below